### ISSUE TITLE
Add Bulgarian translations

### ISSUE THREAD

**Issue by suricactus** [squidfunk/mkdocs-material#2236]
## Instructions

1. Check, if your language is already available: [here](https://bit.ly/33vFDD0)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "bg",
  "direction": "ltr",
  "clipboard.copy": "Копирай",
  "clipboard.copied": "Копирано",
  "edit.link.title": "Редактирай тази страница",
  "footer.previous": "Предишна",
  "footer.next": "Следваща",
  "footer.title": "Долен колонтитул",
  "header.title": "Горен колонтитул",
  "meta.comments": "Коментари",
  "meta.source": "Код",
  "nav.title": "Навигация",
  "search.config.lang": "bg",
  "search.config.pipeline": "trimmer, stopWordFilter",
  "search.config.separator": "[\s\-]+",
  "search.placeholder": "Търси...",
  "search.reset": "Изчисти",
  "search.result.placeholder": "Започнете да пишете, за да търсите",
  "search.result.none": "Няма резултати",
  "search.result.one": "1 резултат",
  "search.result.other": "# резултата",
  "search.result.more.one": "още 1 на тази страница",
  "search.result.more.other": "още # на тази страница",
  "skip.link.title": "Към съдържанието",
  "source.link.title": "Към хранилището",
  "source.revision.date": "Последна промяна",
  "tabs.title": "Табове",
  "toc.title": "Съдържание"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks for providing! It'll be part of the next release.

**Comment by squidfunk** [comment]
Added in ee4efb4f.

**Comment by squidfunk** [comment]
Released as part of 6.2.6.
